app/furniture/marketplace/notification_methods/index.html.erb

```diff
@@ -1,21 +1,29 @@
 <%- breadcrumb(:marketplace_notification_methods, marketplace) %>
-
 <section class="mt-3">
   <main>
-
     <%= render Marketplace::NotificationMethodComponent.with_collection(notification_methods) %>
-
-  </main>
-
-  <div class="text-center mt-3">
-    <%- notification_methods = marketplace.notification_methods.new %>
-    <%- if policy(notification_methods).create? %>
-      <%= render ButtonComponent.new(
-        label: "#{t('marketplace.notification_methods.new.link_to')} #{t('icons.new')}",
-        title: t('marketplace.notification_methods.new.link_to'),
-        href: marketplace.location(:new, child: :notification_method),
-        method: :get,
-        scheme: :secondary) %>
+    <div class="text-center mt-3">
+      <%- notification_methods = marketplace.notification_methods.new %>
+      <%- if policy(notification_methods).create? %>
+        <%= render ButtonComponent.new(
+          label: "#{t('marketplace.notification_methods.new.link_to')} #{t('icons.new')}",
+          title: t('marketplace.notification_methods.new.link_to'),
+          href: marketplace.location(:new, child: :notification_method),
+          method: :get,
+          scheme: :secondary) %>
+      <%- end %>
+    </div>
+    <%= render CardComponent.new(dom_id: dom_id(notification_method), classes: "flex flex-col justify-between gap-y-2 w-full") do %>
+      <details <%= "open" if !marketplace.square_order_notifications_enabled? %>>
```

Comment:
I didn't think `<details>` could hold a `<form>`! But it makes sense that they can!

Reply:
Yeah. This was my clever way of avoiding having to render out to a separate page like the Marketplace `:edit` route. It's a nice stop gap before this lives in it's own `Utility`.